skimage/feature/brief.py

```diff
@@ -64,18 +64,18 @@ class BRIEF(DescriptorExtractor):
     >>> import numpy as np
     >>> square1 = np.zeros((8, 8), dtype=np.int32)
     >>> square1[2:6, 2:6] = 1
-    >>> square1
+    >>> square1  # doctest: +SKIP
```

Comment:
I think you should get linux setup, and run tehse doctests on linux. Linux will be able to handle the doctests and all.

Reply:
@Borda I think `conftest.py` file is not being discovered by pytest on Windows. In that file we force numpy to use legacy printoptions. Therefore, the builds expect the newer notation (that is numerically equivalent, but visually different).

Reply:
@hmaarrfk on Azure you mean? I'm not sure if anyone from the core team is willing to switch from Travis to Azure at the moment. Of course, it shouldn't be an issue to have a Linux instance running, but I really want to have Windows build with the full coverage.
Unfortunately, googling "pytest, windows, conftest" doesn't yield any info. I'm going to check if forcing conftest could be done with the env.variables...

Reply:
We could also just test the doctests on numpy 1.14 and higher

or just bunp the minimum reqs all the way up to 1.14 ;)

Reply:
@soupault, it might be worthwhile too post an issue on pytest's github page. I couldn't figure this out either, it could very well be a bug on windows.

I understand that you want full test coverage on windows, but I would argue that having the doctests be readable is more important. I've often seen my comments copied verbatim in other's code. I wouldn't watn to pollute their dropbox with # doctest: +SKIP   ;)